src/streamlink_cli/output.py

```diff
@@ -143,8 +143,11 @@ def _create_arguments(self):
 
         # player command
         if is_win32:
-            return cmd + " " + subprocess.list2cmdline(extra_args) + " " + args
-        return shlex.split(cmd) + extra_args + shlex.split(args)
+            if is_py2:
+                return unicode.join( u'', [ cmd, " ", subprocess.list2cmdline(extra_args).decode('utf-8'), " ", args ] )
```

Comment:
Is this equivalent to 

```python
  u''.join([cmd, " ", subprocess.list2cmdline(extra_args).decode('utf-8'), " ", args])
```

Reply:
I'm not sure. I was having problems concatting the string and found `unicode.join` as a suggestion. I'll use ` u''.join([cmd, " ", subprocess.list2cmdline(extra_args).decode('utf-8'), " ", args])` if it works and is preferrable

Reply:
We should add a util method for this, something like
```python
def maybe_decode(text, encoding="utf8"):
    if is_py2 and isinstance(text, str):
        return text.decode(encoding)
    else:
        return text
```

Then you could just do 

```python    
return u' '.join([cmd, maybe_decode(subprocess.list2cmdline(extra_args), "utf8"), args])
```

Reply:
And the reverse, `maybe_encode` :)